extension/e2e-tests/sendPayment.test.ts

```diff
@@ -1,8 +1,8 @@
 import { test, expect, expectPageToHaveScreenshot } from "./test-fixtures";
 import { loginAndFund, loginToTestAccount, PASSWORD } from "./helpers/login";
+import { TEST_TOKEN_ADDRESS } from "./helpers/test-token";
 
 test("Send XLM payment to G address", async ({ page, extensionId }) => {
-  test.slow();
```

Comment:
is there a specific reason to drop this? if I recall, it just extends the wait time of a test. I believe I had added this in the event of slow responses from Horizon

Reply:
I just didn't observe that it added any longer of a timeout but no qualms with restoring, will add back.

Reply:
Cool - once we stub out external calls, we shouldn't need this anymore